dev/com.ibm.ws.security.acme/src/com/ibm/ws/security/acme/internal/exceptions/CertificateRenewRequestBlockedException.java

```diff
@@ -0,0 +1,75 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.security.acme.internal.exceptions;
+
+import com.ibm.websphere.ras.annotation.Trivial;
+import com.ibm.ws.security.acme.AcmeCaException;
+
+/**
+ * Exception that is thrown when a certificate renew request occurs too
+ * soon after the prior request.
+ */
+@Trivial
+public class CertificateRenewRequestBlockedException extends AcmeCaException {
+
+	private static final long serialVersionUID = -2381844611991560528L;
+	
+	private long timeLeft = -1;
```

Comment:
Probably should define whether this in ms or s or whatever unit. The set / get methods should be updated as well to make it clear.

Reply:
Updated